src/main/java/org/apache/commons/imaging/formats/tiff/datareaders/DataReaderStrips.java

```diff
@@ -166,7 +166,7 @@ private void interpretStrip(
         // this logic will handle all cases not conforming to the
         // special case handled above
 
-        try (final BitInputStream bis = new BitInputStream(new ByteArrayInputStream(bytes), byteOrder)) {
+        try (BitInputStream bis = new BitInputStream(new ByteArrayInputStream(bytes), byteOrder)) {
```

Comment:
Why did you need to remove the `final` here?

Reply:
This is one of the changes I made based on the PMD recommendations.  Would it be better if I put the "final" modifier back in?

Reply:
The less differences in the pull request that are not directly related, the easier it is to review the changes IMO. But if you changed only one or two places that should be fine, otherwise it's better to send the PMD changes in another PR.